src/Mod/TechDraw/App/DrawViewSymbol.cpp

```diff
@@ -107,7 +114,13 @@ void DrawViewSymbol::onChanged(const App::Property* prop)
                 }
             }
             else {
-                Base::Console().Warning("DrawViewSymbol:onChanged - SVG for Symbol is not a valid document\n");
+                Base::Console().Warning("DVS::onChanged - %s - SVG for Symbol is not valid. See log.\n");

```

Comment:
@WandererFan, this change by you from a while ago adds a `%s` placeholder, but does not passes any variable for it, leading to garbage in the output:

```
Wrn: DVS::onChanged - ó^O^^úUH<89>åSH<81>ì - SVG for Symbol is not valid. See log.
```

Did you maybe intend to pass `getNameInDocument()` here? The same thing below for `execute()`.

Reply:
wandererfan has left the FC community: https://forum.freecadweb.org/viewtopic.php?f=35&t=51233